### ISSUE TITLE
Getting Fatal Error while Executing Xgboost in R

### ISSUE THREAD

**Issue by NayanaNMurthy** [CogComp/lbjava#97]
I am using Xgboost function in R for building model file.

For around 1000 dataset, the function is working correctly.

When I increased the size of dataset to around 300k with 400 features

I am getting fatal error in R, while executing below line:

fit <- xgboost(data=traindata,label=NULL,verbose = 2, nthread = 2,     eta=0.3,nround = 10,objective = "multi:softmax",num_class = length(names) + 1,silent=1)

note: traindata is of type xgb.matrix

I also tried executing the same by varying parameters :eta and nrounds, but still getting fatal error and R session is aborting

>sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

locale:
[1] LC_COLLATE=English_India.1252  LC_CTYPE=English_India.1252  
 LC_MONETARY=English_India.1252

[4] LC_NUMERIC=C                   LC_TIME=English_India.1252    

 attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] tools_3.3.1
I also executed the same in RGui(console), there also I got runtime error

What is the reason for this issue?

**Comment by danyaljj** [comment]
This has anything to do with Lbjava?

**Comment by NayanaNMurthy** [comment]
Thank you for the Response

But How Lbjava is Related to XGboos  ?
